modules/apps/object/object-dynamic-data-mapping-storage/src/main/java/com/liferay/object/dynamic/data/mapping/storage/internal/ObjectDDMStorageAdapter.java

```diff
@@ -49,8 +88,42 @@ public DDMStorageAdapterGetResponse get(
 			DDMStorageAdapterGetRequest ddmStorageAdapterGetRequest)
 		throws StorageException {
 
+		DDMForm ddmForm = ddmStorageAdapterGetRequest.getDDMForm();
+
+		ObjectEntry objectEntry;
+
+		try {
+			objectEntry = _objectEntryManager.getObjectEntry(
+				_getDTOConverterContext(
+					ddmStorageAdapterGetRequest.getPrimaryKey(), null,
+					ddmForm.getDefaultLocale()),
+				ddmStorageAdapterGetRequest.getPrimaryKey());
+		}
+		catch (Exception exception) {
+			throw new StorageException(exception);
+		}
+
+		Map<String, Object> values = objectEntry.getProperties();
+
+		DDMFormValues ddmFormValues = _ddmFieldLocalService.getDDMFormValues(
+			ddmForm, (Long)values.get("ddmStorageId"));
+
+		for (DDMFormFieldValue ddmFormFieldValue :
+				ddmFormValues.getDDMFormFieldValues()) {
+
+			DDMFormField ddmFormField = ddmFormFieldValue.getDDMFormField();
+			Value value = new LocalizedValue(ddmForm.getDefaultLocale());
+
+			value.addString(
+				ddmForm.getDefaultLocale(),
```

Comment:
Just to confirm my understanding, @gabrielwas, for this pull we're only considering the default language, right?

Reply:
Right! For the next steps, we should see how localization works with objects.